js/build_jsep.bat

```diff
@@ -17,7 +17,7 @@ set BUILD_DIR=%ROOT%build_jsep
 :arg1
 if ["%~1"]==["d"] (
     set CONFIG=Debug
-    set CONFIG_EXTRA_FLAG=--enable_wasm_debug_info --enable_wasm_profiling
+    set CONFIG_EXTRA_FLAG=--enable_wasm_debug_info --enable_wasm_profiling --cmake_extra_defines onnxruntime_ENABLE_WEBASSEMBLY_OUTPUT_OPTIMIZED_MODEL=1
```

Comment:
Is it possible to add `--enable_webassembly_output_optimized_mode` command line arguments to build.py?
Does one still need to use session option to specify the output file the dump optimized model?

Reply:
This will break unittest.

In unittest, there are some test cases that optimize the model and check the output. Because all these operations are performed in virtual FS, it actually works. However, if we use this flag, the optimized model will be outputed into Blob, so the later steps will fail because of file missing in virtual FS.

We can disable those test cases in WebAssembly build, but that's going to be one more thing to maintain in future.

Reply:
Thanks for clarifying.